backend/sirius-web-diagrams/src/main/java/org/eclipse/sirius/web/diagrams/components/EdgeComponent.java

```diff
@@ -118,7 +122,7 @@ public Element render() {
                                     .sourceId(sourceId)
                                     .targetId(targetId)
                                     .style(style)
-                                    .routingPoints(List.of())
+                                    .routingPoints(optionalPreviousEdge.map(Edge::getRoutingPoints).orElseGet(()-> edgeRoutingPointsProvider.getRoutingPoints(sourceNode, targetNode)))
```

Comment:
Extract that in a variable too, that's too complex

Reply:
see 0d825fc825323b2902b76c564e851eec89a13e82